tests/python/test_python.py

```diff
@@ -0,0 +1,32 @@
+import taichi as ti
+ti.init()
+
+
+@ti.host_arch_only
+def test_func_closure():
```

Comment:
The filename `test_python.py` is confusing. Could you please describe what this test does more concisely?